ui/src/meta.tsx

```diff
@@ -113,6 +123,10 @@ interface State {
   scripts?: Script[]
   /** Javascript code to execute on this page. */
   script?: InlineScript
+  /** CSS to be inlined into this page. */
+  style?: S
```

Comment:
```suggestion
  stylesheet?: InlineStylesheet
```

`InlineStylesheet` (consistent with InlineScript) can contain `content` and `media?` (for consistency with `Stylesheet.media`).